src/main/java/com/labsynch/labseer/service/LsThingServiceImpl.java

```diff
@@ -1026,8 +1026,12 @@ public LsThing findParentByBatchEquals(LsThing batch) {
 			String searchQuery) {
 		return findLsThingsByGenericMetaDataSearch(searchQuery, null);
 	}
-	
 
+	@Override
+	public Collection<LsThing> findLsThingsByGenericMetaDataSearch(
+			String searchQuery, String lsType) {
+		return findLsThingsByGenericMetaDataSearch(searchQuery, null, null);
```

Comment:
This doesn't make sense. It's taking in a `searchQuery` and `lsType`, then completely ignoring the `lsType`??

Reply:
Yeah, thats wrong, i'll fix it.